src/atoms/Card/Card.css

```diff
@@ -1,4 +1,5 @@
 .Card {
   border: 1px solid var(--color-gray);
   padding: 1.5rem 2rem;
-}
\ No newline at end of file
+  background-color: white;
```

Comment:
I can't recall who made this change. It's a good change to make, but because it's not relevant to what the branch is doing we need to either call this change out in the description or move it to another branch with related work.